README.md

```diff
@@ -32,14 +32,14 @@ jobs:
   auto-label:
     runs-on: ubuntu-latest
     steps:
-      - uses: prince-chrismc/label-merge-conflicts-action@v1
+      - uses: prince-chrismc/label-merge-conflicts-action@v1 # Replace this with the latest version
         with:
           conflict_label_name: "has conflict"
           github_token: ${{ secrets.GITHUB_TOKEN }}
           # These are optional incase you need to adjust for the limitations described below
           max_retries: 5
           wait_ms: 15000
-          detect_merge_changes: false # or true to handle as conflicts
+          mergeable_only: false # or true to unlabel as soon as all the merge requirements are fulfilled
```

Comment:
I think this suffers from the same problem as "merge changes"

It's supper specific to the GitHub API language and I suspect most consumer would know what it means